SettingsAPI.go

```diff
@@ -42,6 +42,10 @@ func (settings *Settings) Edit(ctx *aero.Context, key string, value reflect.Valu
 		}
 
 		return true, nil
+
+	case "CalendarSettings.ShowUserList":
```

Comment:
I don't think this is required here.
Edit is for on-edit hooks that need to be applied on save.
A simple bool value doesn't need to be handled here.